### DISCUSSION TITLE
old phone DB

### DISCUSSION THREAD

**Discussion by Still15** [jamulussoftware/jamulus#3105] [General]
I had sony ericsson w580i long time ago. There was cool bluetooth request sound.  And for a long time i trying to find it and finaly found it on [your website?].
I'd like to say thank you/
![image](https://github.com/jamulussoftware/jamulus/assets/45699515/0798eb21-f51b-4066-9678-ecf655f2c8b9)

**Comment by mcfnord** [comment]
angels watch over us!

**Comment by ann0see** [comment]
Not sure what you mean? I think it's another website. 

Probably it's a server owner who deserves the credit, not us :-).

https://mastodonmusic.social/@Onj

http://www.andrelouis.com/socials

Could be related to the person who is hosting this.

**Reply by Still15** [reply]
I scrolled back to parettial directory and fined page with link to your project. 
I'm not a programmer at all and don't know how to scan web site for all subdomains and hosts owner.

**Reply by ann0see** [reply]
No problem. I just entered the domain from your screenshot and came to the page of the person I mentioned above. 

I think he's the person who deserves credit.
